### PR TITLE
feat(http): Add `Alt‑Svc` header

### PR DESCRIPTION
**Pull request by ExE-Boss**
See&nbsp;[bug&nbsp;1003448](https://bugzil.la/1003448) and&nbsp;[bug&nbsp;1113790](https://bugzil.la/1113790) for&nbsp;details.

review?(@Elchi3, @vinyldarkscratch)

### CODE REVIEW

File: http/headers/alt-svc.json [hunk 1]
```diff
@@ -0,0 +1,105 @@
+{
+  "http": {
+    "headers": {
+      "Accept": {
```
Comment (Elchi3) [inline http/headers/alt-svc.json]:
```suggestion
      "Alt-Svc": {
```

Comment (ExE-Boss) [reply http/headers/alt-svc.json]: We should really add tests to prevent mistakes like this.

File: http/headers/alt-svc.json [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+              "version_added": true
+            },
+            "safari": {
+              "version_added": null
+            },
+            "safari_ios": {
+              "version_added": null
+            },
+            "samsunginternet_android": {
+              "version_added": null
```
Comment (vinyldarkscratch) [inline http/headers/alt-svc.json]: Since Chrome supports it, let’s set this to “true”.  ;)

File: http/headers/alt-svc.json [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+            },
+            "edge_mobile": {
+              "version_added": null
+            },
+            "firefox": [
+              {
+                "version_added": "38"
+              },
+              {
+                "version_added": "37",
```
Comment (vinyldarkscratch) [inline http/headers/alt-svc.json]: I'm thinking that maybe we should add some `version_removed` statements in these blocks.

**Pull request merged**